hy/contrib/multi.hy

```diff
@@ -21,9 +21,7 @@
 ;; DEALINGS IN THE SOFTWARE.
 
 (import [collections [defaultdict]]
-        [hy.models.expression [HyExpression]]
-        [hy.models.list [HyList]]
-        [hy.models.string [HyString]])
+        [hy [HyExpression HyList HyString]])
```

Comment:
I'm confused by this. Shouldn't it be `[hy.models [HyExpression HyList HyString]]`? And yet the tests pass. Are we not testing this?